core/renderers/common/constants.js

```diff
@@ -702,6 +660,87 @@ Blockly.blockRendering.ConstantProvider.prototype.dispose = function() {
   }
 };
 
+/**
+ * Get constants related to text sizing for fields.  All values must be
+ * provided.
+ * In general, changing one of these constants means you should change all of
+ * them.
+ * @return {{
+ *  fontSize: number,
+ *  height: number,
+ *  weight: string,
+ *  family: string,
+ *  baselineY: number,
+ *  baselineCenter: boolean,
+ *  yOffset: number
+ * }} An object containing information about text sizing for fields.
+ * @protected
+ */
+Blockly.blockRendering.ConstantProvider.prototype.getTextConstants_ = function() {
+  return {
+    fontSize: 11,
+    height: 16,
+    weight: 'normal',
```

Comment:
It's hard to tell here what weight and height refer to without having it prefixed with ``font``.